### PR TITLE
Add settings panel

### PR DESCRIPTION
**Pull request by deansallinen**
Adds settings button to header (when no back action is present)

Settings menu includes:
- Language
- Theme
- Animations (enabled/disabled)
- About page with version information etc.
- Github link

### CODE REVIEW

**Comment by aaroncox** [comment]
@deansallinen Merging in the status message PR to dev caused some conflicts here. Mind resolving those (since you'll know better which is new) so we can push this into dev as well?

File: src/ui/components/Modal.svelte [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
     // Perform work required to cancel request
     function cancelRequest() {
+        console.log($props)
```
Comment (aaroncox) [inline src/ui/components/Modal.svelte:24]: A wild console.log appears!

File: src/ui/settings/About.svelte [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    import BodyText from '../components/BodyText.svelte'
+    import BodyTitle from '../components/BodyTitle.svelte'
+    import Link from '../components/Link.svelte'
+    import {getThemedLogo} from 'src/lib/utils'
+    const {t} = getContext<i18nType>('i18n')
+
+    const wharfLogo = {
+        name: 'Wharf',
+        logo: {
+            light: 'https://raw.githubusercontent.com/wharfkit/assets/master/Logo/Svg/Wharf_logo_bright_horizontal_vector_no_bg.svg',
```
Comment (aaroncox) [inline src/ui/settings/About.svelte:16]: We have these on our CDN, and could avoid using Github.

- https://assets.wharfkit.com/wharf-brand-assets/logo/svg/wharf-logo-dark-horizontal-vector-no-bg.svg
- https://assets.wharfkit.com/wharf-brand-assets/logo/svg/wharf-logo-bright-horizontal-vector-no-bg.svg

File: src/ui/components/Modal.svelte (hunk 1 above)
Comment (deansallinen) [reply src/ui/components/Modal.svelte:24]: Removed!

**Pull request merged**